### PR TITLE
update for MathJax v2.6.0

### PR DESCRIPTION
**Pull request by pkra**
- fix SVG config names
- update CommonHTML section
- add previewHTML and plainSource sections
- add a11y section
- add misc items
- fix typo

### CODE REVIEW

**Comment by pkra** [comment]
Ready for review.

File: Gruntfile.js [hunk 1]
```diff
@@ -209,23 +209,37 @@ module.exports = function(grunt) {
...
         "extensions/CHTML-preview.js",
         "unpacked/extensions/CHTML-preview.js"
       ],
+      previewHtmlOutput: [
+        "jax/output/PreviewHTML",
+        "unpacked/jax/output/PreviewHTML",
+        "extensions/fast-preview.js",
+        "unpacked/extensions/fast-preview.js"
+        "extensions/CHTML-preview.js",
+        "unpacked/extensions/CHTML-preview.js"
```
Comment (dpvc) [inline Gruntfile.js:233]: Are these lists of files to be _kept_ or ones to be _removed_?  Note that `CHTML-preview.js` is a legacy file that is never needed for new installations (it is only there for people who included it before CommonHTML was finished and we renamed the old CommonHTML to PreviewHTML).  The old `CHTML-preview.js` is now `fast-preview.js`.  So you don't needed to include `CHTML-preview.js` here or above, I think, if these are files to be included.  If they are files to be excluded, then you should always remove `CHTML-preview.js`.

Also, is the `previewHtmlOutput` block used for including fast preview in the build, or for using PreviewHTML as the output renderer?  If the latter, you don't need `fast-preview.js` here.

Comment (pkra) [inline Gruntfile.js:233]: > Are these lists of files to be kept or ones to be removed? 

Removed. (It's mentioned in the readme.)

>  If they are files to be excluded, then you should always remove CHTML-preview.js.

People may have used it in the past and want to keep it.

> Also, is the previewHtmlOutput block used for including fast preview in the build, or for using PreviewHTML as the output renderer? 

Both.

Comment (dpvc) [inline Gruntfile.js:233]: OK, then I guess it is ready to merge.

Comment (pkra) [inline Gruntfile.js:233]: Thanks!

**Pull request merged**